tensorflow/lite/micro/tools/make/targets/cortex_m_generic_makefile.inc

```diff
@@ -115,7 +115,7 @@ ifeq ($(TOOLCHAIN), armclang)
   MICROLITE_LIBS := $(filter-out -lm,$(MICROLITE_LIBS))
 
 else ifeq ($(TOOLCHAIN), gcc)
-  TARGET_DEFAULT_TOOLCHAIN_ROOT := $(MAKEFILE_DIR)/downloads/gcc_embedded/bin/
+  TARGET_DEFAULT_TOOLCHAIN_ROOT := ${TENSORFLOW_ROOT}/$(MAKEFILE_DIR)/downloads/gcc_embedded/bin/
```

Comment:
ah, just noticed this.

This should be:
```
${TENSORFLOW_ROOT}$(MAKEFILE_DIR)/...
```

i.e. without the / after `TENSORFLOW_ROOT` so that when `TENSORFLOW_ROOT` is empty we still have the correct path.

Reply:
good catch!